src/TKey.cpp

```diff
@@ -158,7 +159,7 @@ void TKey::compile()
     if (mNeedsToBeCompiled) {
         if (!compileScript()) {
             if (mudlet::debugMode) {
-                TDebug(Qt::white, Qt::red) << "ERROR: Lua compile error. compiling script of key binding:" << mName << "\n" >> 0;
+                TDebug(Qt::white, Qt::red) << "ERROR: Lua compile error. compiling script of key binding:" << mName << "\n" >> mpHost;
```

Comment:
If re require host to be added to debug message, shouldn't it be enforced by param? This way we have to know convention and "remember".

Reply:
I am not entirely sure whether a literal `0` in the source code might not be capable of being converted to a `nullptr` - I think not in modern C++17 code (as part of the purpose of `nullptr` is to avoid it being mistaken as something else) and I *think* the compiler will start to moan at you if you do not change that `0`; but, as someone who come from a C coding background, I am aware that C89, for instance, is a bit more lax about this sort of thing...

Reply:
The original proposal paper might clarify things: http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2007/n2431.pdf

Reply:
Ok, wasn't sure so thought that it is better to ask than keep quiet :)